### PR TITLE
Containers should not get inheritable caps by default

### PR DESCRIPTION
**Pull request by rhatdan**
When I launch a container with --userns=keep-id the rootless processes
should have no caps by default even if I launch the container with
--privileged.  It should only get the caps if I specify by hand the
caps I want leaked to the process.

Currently we turn off capeff and capamb, but not capinh.  This patch
treats capinh the same way as capeff and capamb.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Helps fix https://github.com/containers/toolbox/issues/651

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

**Comment by baude** [comment]
lgtm

**Comment by ashley-cui** [comment]
Unhappy tests :(

**Comment by mheon** [comment]
I think this one needs further discussion. It is 100% not a fix for the reported issue, but it could still be a good idea.

**Comment by mheon** [comment]
Good candidate for watercooler.

File: libpod/oci.go [hunk 1]
```diff
@@ -151,8 +151,6 @@ type OCIRuntime interface {
 type ExecOptions struct {
 	// Cmd is the command to execute.
 	Cmd []string
-	// CapAdd is a set of capabilities to add to the executed command.
-	CapAdd []string
```
Comment (mheon) [inline libpod/oci.go:155]: I think you need to retain this - `podman exec --privileged` can be run in containers that are not privileged.

**Pull request merged**